src/Sql/Sql/ManagedInstance/Cmdlet/NewAzureSqlManagedInstance.cs

```diff
@@ -149,6 +150,23 @@ public class NewAzureSqlManagedInstance : ManagedInstanceCmdletBase
         [PSArgumentCompleter(Constants.CollationSqlLatin1, Constants.CollationLatin1)]
         public string Collation { get; set; }
 
+        /// <summary>
+        /// Gets or sets whether or not the public data endpoint is enabled.
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "Whether or not the public data endpoint is enabled for the instance.")]
+        public SwitchParameter PublicDataEndpointEnabled { get; set; }
+
+        /// <summary>
+        /// Gets or sets connection type used for connecting to the instance.
+        /// Possible values include: 'Proxy', 'Redirect', 'Default'
```

Comment:
I only see two values on the argument completer - can you not explicitly set default?

Reply:
Updated.